README.md

```diff
@@ -160,3 +189,32 @@ This file will be ignored if a package specific template already exists, searchi
 $(PackagePath)/spec.template
 $(ProjectPath)/config/specTemplate.spec
 ```
+
+## Examples
+### Multiple simultaneous developments
+If developing, e.g,. both `xhal` and `ctp7_modules` simultaneously:
+
+#### Create a local `xhal` installation
+```sh
+cd /path/to/xhal
+INSTALL_PREFIX=/path/to/local/installation make install
+```
+
+#### Use local `xhal` installation when compiling `ctp7_modules`
+##### Set up the local `PETA_STAGE` area
+If working with multiple target boards, repeat for each
+```sh
+ln -ns /opt/gem-peta-stage/ctp7/usr /path/to/local/installation/opt/gem-peta-stage/ctp7/
+ln -ns /opt/gem-peta-stage/ctp7/lib /path/to/local/installation/opt/gem-peta-stage/ctp7/
+## link in any system installed packages you're not modifying
+ln -ns /opt/gem-peta-stage/ctp7/mnt/persistent/reedmuller /path/to/local/installation/opt/gem-peta-stage/ctp7/mnt/persistent/
+```
+##### Move to `ctp7_modules` work area
+```sh
+cd /path/to/ctp7_modules
+export PETA_STAGE=/path/to/local/installation/opt/gem-peta-stage
```

Comment:
Is this targeting legacy only releases? In the templated version of the `ctp7_modules` we are getting rid of `$PETA_STAGE` and exporting it here doesn't seems to be a proper move...

Reply:
The hotfix is general, the documentation update is for the new framework.
Point me to an issue where removal of `PETA_STAGE` is discussed and I can rework this, however, `PETA_STAGE` is currently used by **all** templated compatible code building (it is just not required to be set by the developer except in certain cases)